lib/roger/resolver.rb

```diff
@@ -176,7 +176,7 @@ def sanitize_name(name, prefer = nil)
       path = name.to_s
 
       # If it's a directory append "index"
-      path = File.join(path, "index") if File.directory?(name)
+      path = File.join(path, "index") if File.directory?(name) && Dir.glob(name + ".*").none?
```

Comment:
Agreed, its leaky. But figured since its only evaluated when name is a directory it doesn't hurt.

No I want to check for this: 

```
-html
  - bla/
  - bla.html.erb
```